### PR TITLE
Set up "skeleton" of happy path for Individuals

### PR DESCRIPTION
**Pull request by irisfaraway**
https://eaflood.atlassian.net/browse/WC-148

This story covers setting up a "skeleton" of the happy path for those who would select Individuals on the select org type page. Pages in the skeleton will mostly be blank – just a title and continue button.

This is so we can set up how users move between the pages, and also so we can start testing the process as a whole.

As we continue to work on the stories, these pages will be fleshed out with the proper forms.

### CODE REVIEW

**Comment by irisfaraway** [comment]
@Cruikshanks Think you could have a look at this? If you're happy with this format I'll crack on with the other business types.

Copy needs work but I think we'll get to that when we actually make the pages. For now, we can at least differentiate between the different business types.

I did somehow muck up the `git mv` for two renamed files but the others should be OK 😅

File: config/locales/forms/registration_number_forms/en.yml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 en:
   registration_number_forms:
     new:
-      heading: What's the registration number of the company?
+      heading_limitedCompany: What's the registration number of the company?
+      heading_soleTrader: What's the registration number of the individual?
```
Comment (Cruikshanks) [inline config/locales/forms/registration_number_forms/en.yml]: We should never have a need for this as an individual should never go down a path that requests a registration number.

I'd rather take this out so if we have something in the sole trader journey that relies on this translation existing we know about it.

On a side note, do we have the app configured to raise an exception in dev and test if a translation is missing?

File: db/seeds.rb [hunk 2]
```diff
@@ -86,4 +87,79 @@
       confirmed: "no"
     }
   )
+
+  # Sole trader
```
Comment (Cruikshanks) [inline db/seeds.rb]: Reckon this is fine for this PR, but we need to think about a different approach going forward especially as @timstone is now busy putting together his list of registrations he'd like to see seeded!

File: config/locales/forms/registration_number_forms/en.yml (hunk 1 above)
Comment (irisfaraway) [reply config/locales/forms/registration_number_forms/en.yml]: You're completely right. I was thinking it was weird that there was a reg number page for individuals but I must have misread the wireframe/journey map. Will sort this out!

And no, we don't have that configured – I'll look into it.

Comment (Cruikshanks) [reply config/locales/forms/registration_number_forms/en.yml]: It's a little old, but I think it covers our version of rails https://robots.thoughtbot.com/foolproof-i18n-setup-in-rails

Comment (irisfaraway) [reply config/locales/forms/registration_number_forms/en.yml]: Both these issues should now be fixed 👍

File: db/seeds.rb (hunk 2 above)
Comment (irisfaraway) [reply db/seeds.rb]: Agreed, this could get messy pretty quickly – should we have a story to look into how to handle seeding in a sustainable way?

Comment (Cruikshanks) [reply db/seeds.rb]: Agreed and done! Implement sustainable method for seeding the database

**Pull request merged**